formatter_test.go

```diff
@@ -93,3 +93,159 @@ func TestFormatDiff(t *testing.T) {
 	checkOK(map[string]interface{}{"a": 1}, map[string]interface{}{})
 	checkOK([]interface{}{"a"}, []interface{}{})
 }
+
+func TestFormatFailure_Actual(t *testing.T) {
```

Comment:
Nitpick: in this project underscores are not used in test names

Reply:
ok, so it should be `TestFormatFailureActual`?

Reply:
updated https://github.com/gavv/httpexpect/pull/202/commits/07e108659e713e09854698571aff472eab9fe7fa

Reply:
Yep